doc/user-manual/language/telescopes.lagda.rst

```diff
@@ -2,14 +2,104 @@
   ::
   module language.telescopes where
 
+  open import Agda.Builtin.Bool
+  open import Agda.Builtin.Nat
+
+  postulate
+    List : Set → Set
+    _++_ : {A : Set} → List A → List A → List A
+    Vec : Set → Nat → Set
+    IsTrue : Bool → Set
+    Monoid : Set → Set
+    NonZero : Nat → Set
+
+  record _×_ (A B : Set) : Set where
+    constructor _,_
+    field
+      fst : A
+      snd : B
+
 .. _telescopes:
 
 **********
 Telescopes
 **********
 
+A telescope is a non-empty sequence of variable bindings annotated by their types,
+with each variable surrounded by parentheses.
```

Comment:
Shouldn't this also mention the possibility of leaving out the types? 
(see below)

Reply:
It's mentioned below, do you think it's useful to already mention it here? Morally speaking these variables still have a type annotation, it's just inferred by Agda rather than explicit in the syntax.

Reply:
Ok!